### ISSUE TITLE
separate compact_segments() into own command?

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#2294]
We've seen at misc. places (in tracebacks), that sometimes borg crashes in compact_segments, after the main task is already done.

E.g.:
- borg check #2137 
- borg delete --force (comment about crash in src)

Also, if one e.g. manually does multiple borg operations, running compact N times might be slower (and more annoying as it makes you wait) that running it once at the end, after one is finished with everything else.

**Comment by ThomasWaldmann** [comment]
duplicate of #2195
